File: server/DocumentsKM/Models/Employee.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DocumentsKM.Models
{
    // Сотрудник
    public class Employee
    {
        [Key]
        public Int32 Id { get; set; }

        // Полное имя
        [Required]
        [MaxLength(255)]
        public string Fullname { get; set; }

        // Фамилия
        [Required]
        [MaxLength(80)]
        public string Name { get; set; }

        // Отдел
        [Required]
        [ForeignKey("DepartmentId")]
        public virtual Department Department { get; set; }
        public Int16 DepartmentId { get; set; }

        // Должность
        [ForeignKey("PositionId")]
        public virtual Position Position { get; set; }
        public Int16 PositionId { get; set; }

        [Required]
        public bool IsActive { get; set; }
    }
}

File: client/src/model/Employee.ts
import Department from './Department'

interface IEmployee {
	id: number
	fullname: string
	department: Department
}

class Employee {
	id: number
	fullname: string
	department: Department

	constructor(obj?: IEmployee) {
		this.id = (obj && obj.id) || 0
		this.fullname = (obj && obj.fullname) || ''
		this.department = (obj && obj.department) || null
	}
}

export default Employee
